src/services/pokemon.go

```diff
@@ -0,0 +1,54 @@
+package services
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/vanduc1102/ondemand-go-bootcamp/src/models"
+	"github.com/vanduc1102/ondemand-go-bootcamp/src/utils"
+)
+
+func FindById(id int) (*models.Pokemon, error) {
```

Comment:
Remember to add a comment as a good practice, as this contributes to the documentation of the code.